packages/utils/src/index.ts

```diff
@@ -1 +1,2 @@
 export { default as unFormatLocaleString } from './unFormatLocaleString';
+export * from './url-ops';
```

Comment:
```suggestion
export { default as isExternalLink } from './isExternalLink';
```
@yauheni-deriv Let's follow the same pattern 🙏🏻 Also not sure why the file name is `url-ops` 🤔

Reply:
This will not be just one function. Planning to move the remaining pure URL functions from @deriv/shared to this file.
Will think about a different name

Reply:
@likhith-deriv Ah so you mean we put all the URL-related utilities in one file, Something like `packages/shared/src/utils/string/string_util.ts`, Right? 🤔 Yeah categorizing the utilities sounds good, Let's discuss this later 🙇🏻